.github/scripts/install_slices/install_slices.py

```diff
@@ -280,6 +293,49 @@ def install_slice(slice: str, arch: str, release: str) -> None:
                 res.stderr.rstrip(),
             )
             sys.exit(res.returncode)
+        # Check if the copyright file has been installed with this slice
+        if list(
```

Comment:
[Nit]: Can we use `any(...)` instead?

Another nit is that it might be easier to read if:
```python
copyright_present = any(filter(
                lambda path: f"/usr/share/doc/{pkg}/copyright" in str(path)
                and path.is_file(),
                pathlib.Path(tmpfs).rglob("*"),
            ))
missing_copyright[slice_name] = !copyright_present
```
but this comes down to personal opinion. If we end up doing this, seeing the logic here and in the main function, it might be easier to use `has_copyright` instead an invert the logic.

Reply:
In fact, now that I re-read this, couldn't we use [os.path.isfile](https://docs.python.org/3/library/os.path.html#os.path.isfile) directly to avoid enumerating all of the files? Or is the copyright sometimes named differently?

Reply:
done in 906a61e

Reply:
> In fact, now that I re-read this, couldn't we use [os.path.isfile](https://docs.python.org/3/library/os.path.html#os.path.isfile) directly to avoid enumerating all of the files? Or is the copyright sometimes named differently?

hum...sounds about right. I'll update it

Reply:
done in https://github.com/canonical/chisel-releases/pull/232/commits/620487cdff3c94f8e38ceb5b0037db4d56fa6c14